CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0]
+RAiDER package was refactored to expose the main functionality as a Python library, including the `prepareWeatherModel` 
```

Comment:
What are the test era5 erai era5t netcdf files added? No reference to it here in the log

Reply:
Those files shouldn't have been changed; updated.